pkg/virt-controller/watch/network.go

```diff
@@ -19,28 +19,20 @@
 package watch
 
 import (
-	k8sv1 "k8s.io/api/core/v1"
+	"encoding/json"
+	"fmt"
 
 	v1 "kubevirt.io/api/core/v1"
 
-	"kubevirt.io/client-go/log"
-
-	"kubevirt.io/kubevirt/pkg/network/namescheme"
 	"kubevirt.io/kubevirt/pkg/network/vmispec"
-	"kubevirt.io/kubevirt/pkg/virt-controller/services"
 )
 
-func calculateDynamicInterfaces(vmi *v1.VirtualMachineInstance, pod *k8sv1.Pod) ([]v1.Interface, []v1.Network, bool) {
+func calculateDynamicInterfaces(vmi *v1.VirtualMachineInstance) ([]v1.Interface, []v1.Network, bool) {
 	vmiSpecIfaces := vmispec.FilterInterfacesSpec(vmi.Spec.Domain.Devices.Interfaces, func(iface v1.Interface) bool {
 		return iface.State != v1.InterfaceStateAbsent
 	})
 	ifacesToHotUnplugExist := len(vmi.Spec.Domain.Devices.Interfaces) > len(vmiSpecIfaces)
 
-	if ifacesToHotUnplugExist && namescheme.PodHasOrdinalInterfaceName(services.NonDefaultMultusNetworksIndexedByIfaceName(pod)) {
-		vmiSpecIfaces = vmi.Spec.Domain.Devices.Interfaces
-		ifacesToHotUnplugExist = false
-		log.Log.Object(vmi).Error("hot-unplug is not supported on old VMIs with ordered pod interface names")
```

Comment:
I'm not really sure what you mean. The condition cannot occur anymore now.

Reply:
I mean to have the log message`hot-unplug is not supported on old VMIs with ordered pod interface names` where possible.
I am asking because it seems that such a scenario can still occur, and I think its worth having such log message.

Reply:
So you mean to add it in the VM controller.

Well, I do not see why we need to bother.
We need to have users who do unplug, on old VM/s and who bother looking into the logs to see why it did not work. Very low probability.
But if @AlonaKaplan also thinks it is useful, I will add it.